linkerd/admin/src/test/scala/io/buoyant/linkerd/admin/names/DelegateApiHandlerTest.scala

```diff
@@ -58,15 +59,15 @@ class DelegateApiHandlerTest extends FunSuite with Awaits {
   }
 
   test("invalid path results in 400") {
```

Comment:
Is it easy to test what happens when we request for a non-delegating interpreter?